sphinx/screens-detail/join-tribe/join-tribe/src/main/java/chat/sphinx/join_tribe/ui/JoinTribeFragment.kt

```diff
@@ -141,35 +141,35 @@ internal class JoinTribeFragment: SideEffectFragment<
         }
     }
 
-    private fun showTribeData(tribe: TribeDto) {
+    private fun showTribeData(viewState: JoinTribeViewState.TribeLoaded) {
         binding.apply {
 
             val owner = viewModel.accountOwnerStateFlow.value
 
             binding.includeTribeMemberInfo.apply {
-                val memberAlias = tribe.myAlias ?: owner?.alias?.value
+                val memberAlias = viewState.myAlias ?: owner?.alias?.value
                 tribeMemberAliasEditText.setText(memberAlias)
 
-                val memberPhotoUrl = tribe.myPhotoUrl ?: owner?.photoUrl?.value
+                val memberPhotoUrl = viewState.myPhotoUrl ?: owner?.photoUrl?.value
                 loadProfileImage(memberPhotoUrl)
             }
 
-            loadTribeImage(tribe)
+            loadTribeImage(viewState.imageUrl)
 
-            textViewTribeName.text = tribe.name
-            textViewTribeDescription.text = tribe.description
-            includeTribePrice.textViewPricePerMessage.text = tribe.price_per_message.toString()
-            includeTribePrice.textViewPriceToJoin.text = tribe.price_to_join.toString()
-            includeTribePrice.textViewAmountToStake.text = tribe.escrow_amount.toString()
-            includeTribePrice.textViewTimeToStake.text = tribe.hourToStake.toString()
+            textViewTribeName.text = viewState.name
+            textViewTribeDescription.text = viewState.description
+            includeTribePrice.textViewPricePerMessage.text = viewState.pricePerMessage.toString()
+            includeTribePrice.textViewPriceToJoin.text = viewState.priceToJoin.toString()
+            includeTribePrice.textViewAmountToStake.text = viewState.escrowAmount.toString()
+            includeTribePrice.textViewTimeToStake.text = viewState.hourToStake.toString()
 
             loadingTribeInfoContent.goneIfFalse(false)
         }
     }
 
-    private fun loadTribeImage(tribe: TribeDto) {
+    private fun loadTribeImage(img: String?) {
         onStopSupervisor.scope.launch(viewModel.mainImmediate) {
-            tribe.img?.let { img ->
+            img?.let { img ->
```

Comment:
utilizing `if (img == null) { ...` here is much more preferred. We should shy away from using `?.let` on immutable properties as it unnecessarily creates a static variable to utilize within the lambda.